recipes/unicorn-binance-rest-api/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "unicorn-binance-rest-api" %}
+{% set version = "1.4.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/unicorn-binance-rest-api-{{ version }}.tar.gz
+  sha256: 51aa63db35cfd55d7766c7f2a54bc4dd7653dbfebde03d2ab6fa1c285daa8321
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - certifi
+    - colorama
+    - cryptography
+    - dateparser
+    - pyopenssl
+    - python >=3.6
+    - regex <=2022.3.2
+    - requests
+    - service_identity
+    - ujson
```

Comment:
Is all of this also required for host? It seems like you only need pip and python and the rest needs to be on run.

Ref: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-rest-api/blob/e46c06cd8470c620188e16f8b29bac59850a937b/setup.py#L57

Reply:
removed

Reply:
I just checked your code and you are importing your module when building it: https://github.com/LUCIT-Systems-and-Development/unicorn-binance-rest-api/blob/e46c06cd8470c620188e16f8b29bac59850a937b/setup.py#L38

This means that you actually do need to import all of the run dependencies in host as well so please add them back for now. It's not really best practise but the quickest solution. :)
For the future, it might be good to remove that and get the version from a __version__ file or something else.

Reply:
I would suggest just putting everything back. That way everything is in sync. You can go and check how many packages are actually imported when building but it seems like speed is of the essence here. :)

Reply:
ah, got it, yes i use it to get the current version for the build :)

ok, i move it back